packages/web-components/examples/codesandbox/components/card-section-offset/cdn.html

```diff
@@ -0,0 +1,109 @@
+<!--
+@license
+
+Copyright IBM Corp. 2020
+
+This source code is licensed under the Apache-2.0 license found in the
+LICENSE file in the root directory of this source tree.
+-->
+
+<html>
+<head>
+  <title>@carbon/ibmdotcom-web-components example</title>
+  <meta charset="UTF-8" />
+  <link rel="stylesheet" href="https://1.www.s81c.com/common/carbon-for-ibm-dotcom/tag/v1/latest/plex.css" />
+  <style>
+    /* Suppress custom element until styles are loaded */
+    dds-card-section-offset:not(:defined) {
+      display: none;
+    }
+  </style>
+  <script type="module" src="https://1.www.s81c.com/common/carbon-for-ibm-dotcom/tag/v1/latest/card-section-offset.min.js"></script>
```

Comment:
@ariellalgilmore can you point to `canary` instead? It would help with our snapshot testing since latest doesn't exist yet, and we have the other codesandbox examples pointing to canary too.

Reply:
updated!